src/compressor/CompressorClient.h

```diff
@@ -0,0 +1,44 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+
+#pragma once
+
+#include <cstdint>
+#include <vector>
+#include "include/buffer_fwd.h"
+#include "CompressorRegistry.h"
+#include "auth/Auth.h"
+
+class CompConnectionMeta;
+class Connection;
+
+class CompressorClient {
+    CompressorRegistry comp_registry;
+public:
+    CompressorClient(CephContext *cct) : comp_registry(cct) { 
```

Comment:
wrong indent. see https://github.com/ceph/ceph/blob/master/CodingStyle#L76. i am not sure if your editor respects the editor variables defined at the beginning of this source file. if it does not, please make sure the indent follows our coding style.